src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -1,16 +1,108 @@
+import javax.swing.UIManager;
+
+import org.apache.commons.httpclient.HostConfiguration;
+import org.pushingpixels.substance.api.skin.SubstanceGraphiteLookAndFeel;
+
+import games.strategy.debug.ClientLogger;
+import games.strategy.debug.ErrorConsole;
+import games.strategy.engine.ClientContext;
+import games.strategy.engine.ClientFileSystemHelper;
+import games.strategy.engine.framework.map.download.MapDownloadController;
+import games.strategy.engine.framework.startup.ui.MainFrame;
+import games.strategy.engine.lobby.server.LobbyServer;
+import games.strategy.triplea.ui.menubar.TripleAMenuBar;
+import games.strategy.ui.SwingAction;
+import games.strategy.util.CountDownLatchHandler;
+import games.strategy.util.EventThreadJOptionPane;
+import games.strategy.util.Version;
 
-/**
- * This class starts and runs the game.
- * <p>
- * This class is compiled to run under older jdks (1.3 at least), and should not do anything more than check the java
- * version
- * number, and then delegate to GameRunner2
- * <p>
- */
 public class GameRunner {
+
+  // not arguments:
+  public static final int PORT = 3300;
+  public static final String LOOK_AND_FEEL_PREF = "LookAndFeel";
+  public static final String DELAYED_PARSING = "DelayedParsing";
+  public static final String CASUALTY_SELECTION_SLOW = "CasualtySelectionSlow";
+  public static final String PROXY_CHOICE = "proxy.choice";
+  public static final String HTTP_PROXYHOST = "http.proxyHost";
+  public static final String HTTP_PROXYPORT = "http.proxyPort";
+  // do not include this in the getProperties list. they are only for loading an old savegame.
+  public static final String OLD_EXTENSION = ".old";
+  // argument options below:
+  public static final String TRIPLEA_GAME_PROPERTY = "triplea.game";
+  public static final String TRIPLEA_SERVER_PROPERTY = "triplea.server";
+  public static final String TRIPLEA_CLIENT_PROPERTY = "triplea.client";
+  public static final String TRIPLEA_HOST_PROPERTY = "triplea.host";
+  public static final String TRIPLEA_PORT_PROPERTY = "triplea.port";
+  public static final String TRIPLEA_NAME_PROPERTY = "triplea.name";
+  public static final String TRIPLEA_SERVER_PASSWORD_PROPERTY = "triplea.server.password";
+  public static final String TRIPLEA_STARTED = "triplea.started";
+  public static final String LOBBY_HOST = "triplea.lobby.host";
+  public static final String LOBBY_GAME_COMMENTS = "triplea.lobby.game.comments";
+  public static final String LOBBY_GAME_HOSTED_BY = "triplea.lobby.game.hostedBy";
+  public static final String LOBBY_GAME_SUPPORT_EMAIL = "triplea.lobby.game.supportEmail";
+  public static final String LOBBY_GAME_SUPPORT_PASSWORD = "triplea.lobby.game.supportPassword";
+  public static final String LOBBY_GAME_RECONNECTION = "triplea.lobby.game.reconnection";
+  // what is the default version of triplea (the one in the "bin" folder)
+  public static final String TRIPLEA_ENGINE_VERSION_BIN = "triplea.engine.version.bin";
+  // proxy stuff
```

Comment:
Another comment that can be wiped out, it's obvious that PROXY_HOST and PROXY_PORT refer to 'proxy stuff'